cocos/spine/skeleton-cache.ts

```diff
@@ -449,7 +450,6 @@ class SkeletonCache {
                 animationCache = new AnimationCache(spData);
                 animationCache._privateMode = this._privateMode;
             }
-            animationCache.init(skeletonInfo, animationName);
```

Comment:
Why doesn't need to call init method now?

Reply:
It will be called outer this block